.github/workflows/stackhpc-all-in-one.yml

```diff
@@ -59,6 +59,10 @@ on:
         description: Whether to perform an upgrade
         type: boolean
         default: false
+      openstack_tests_version:
```

Comment:
nit: this name is a bit deceptive, since it's not really testing openstack itself. It's a bad repo name to begin with though, I definitely see why you called it that. If `stackhpc_openstack_tests_version` fits, I'd use that

Reply:
Would it be a big change to rename the repo?

Reply:
At this point, probably not... might be best to do it now while it's early days